src/components/LoanCards/LoanProgressGroup.vue

```diff
@@ -51,7 +51,7 @@ export default {
 			const formattedTimeLeft = `${this.timeLeft !== '' ? `. ${this.timeLeft}` : ''}`;
 			// Some time left strings already include an exclamation mark
 			const exclamationMark = this.amountLow && !formattedTimeLeft.includes('!') ? '!' : '';
-			return `${formattedMoneyLeft} to go${formattedTimeLeft}${exclamationMark}`;
+			if (!this.numeralLeft.value()) return 'Funded!'; return `${formattedMoneyLeft} to go${formattedTimeLeft}${exclamationMark}`; // eslint-disable-line max-len
```

Comment:
The second `return` should just be moved to a new line for readability.

Reply:
Moved, ty!